rfcs/text/0000-image-base-transcriptomics.md

```diff
@@ -0,0 +1,601 @@
+the state of image-based transcriptomics analyses can be found
+[<span class="underline">here</span>](https://docs.google.com/document/d/18Ve9C_cEn5lI2paivfgTAyjsM62qMyUF_s4mj82l1L0/edit#).
+
+### Data Processing Infrastructure
+
+All *starfish* pipelines share the same inputs and outputs. It should
+therefore be possible to deliver a single adapter WDL that satisfies any
+SpaceTx assay, and a family of subscriptions with minimal inter-class
+changes that specify the particular assay in question.
+
+### Delivery of Output Data to Users
+
+#### Matrices
+
+As mentioned above, image-based transcriptomics output data adhere to
+existing Matrix Service specifications and can be served "as is" for the
+imaging prototype. Additional information on cellular geometries and QC
+will be provided as a supplementary metadata zarr archive.
+
+However, other imaging experiments will likely require a generalization
+of the internal namespace of the DCP. Currently, the DCP standardizes
+around GENCODE gene IDs and HGNC gene symbols. Image based proteomics
+will require protein identifiers such as UNIPROT, and other use cases
+that measure sub-cellular components will likely require more general
+cellular component information (e.g. "endoplasmic reticulum"). These use
+cases could be supported by switching identifiers from, e.g. "ACTB" to
+"HGNC:ACTB" by prepending the namespace of the IDs being used in a given
+assay.
+
+### **Acceptance Criteria**
+
+Except where listed ("responsible team"), the starfish and OME teams
+will do the majority of project management and execution for the
+following deliverables. They will coordinate and collaborate with DCP
+partners ("contact person") to validate design correctness and minimize
+code changes and their impact.
+
+<table>
+<thead>
+<tr class="header">
+<th><strong>Component/Deliverable</strong></th>
+<th><strong>Responsible Team/Service</strong></th>
+<th><strong>No earlier than</strong></th>
+<th><p><strong>Contact</strong></p>
+<p><strong>Person </strong></p></th>
+</tr>
+</thead>
+<tbody>
+<tr class="odd">
+<td>HCA Metadata Draft</td>
+<td>EBI</td>
+<td>Complete</td>
+<td>Z. Perova</td>
+</tr>
+<tr class="even">
+<td>Test Datasets Available</td>
+<td><em>starfish</em></td>
+<td>Complete</td>
+<td>A. Carr</td>
+</tr>
+<tr class="odd">
+<td>Submission Datasets Available</td>
+<td><em>starfish</em></td>
+<td>Q2 2019</td>
+<td>A. Carr</td>
+</tr>
+<tr class="even">
+<td>Analysis Pipelines</td>
+<td>Broad</td>
+<td>Q2 2019</td>
+<td>?</td>
+</tr>
+<tr class="odd">
+<td>Analysis &amp; Adapter Pipelines</td>
+<td>Broad</td>
+<td>Q2 2019</td>
+<td>A. Carr</td>
+</tr>
+<tr class="even">
+<td>Lira subscription</td>
+<td>Broad</td>
+<td>TBD</td>
+<td>S. Ehsan</td>
+</tr>
+<tr class="odd">
+<td>Data Analysis Service <em>starfish</em> Pipeline Tests</td>
+<td>Broad</td>
+<td>Q2 2019</td>
+<td>R. Wang</td>
+</tr>
+<tr class="even">
+<td>Specification for input format</td>
```

Comment:
Does this include appropriate ranges for user submitted parameters that we can use as guardrails or does that fall into the input validation?

Reply:
What would you prefer? In my mind it was in Input validation but I don't have strong preferences.

Reply:
I'm sorry I missed this comment forever ago. I would prefer input validation!